addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2739,6 +2739,194 @@ options.registry.ReplaceImage = options.Class.extend({
     },
 });
 
+options.registry.ConditionalVisibility = options.Class.extend({
+
```

Comment:
Remove this blank line and the blank line between functions. The docstring as separator is enough.